### PR TITLE
Create random_dot_motion.md

### PR DESCRIPTION
**Pull request by bvikram03**

### CODE REVIEW

File: tasks/random_dot_motion.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+
+# Procedure
+## Steps
+> There are three difficulty levels. The three difficulty levels were characterized by
+different proportions of correlated dots, pc = 50%, 30%, and 5%. In addition, the deviation was set to 0° in difficulty level 1 and to 45° in
+difficulty levels 2 and 3.
+
+ > There are 2 versions of the study
+ > Version 1: Individuals were paired with the same partner for 15 consecutive rounds. In each round, the individual made a guess, then looked at their partner's estimate, and finally revised their guess. Partners always had the same correct angle programmed into the dots, but the difficulty level could vary by trial between the pair. 
+ > Version 2: A group of 6 participants (A, B...F) are connected along a chain. After each initial guess, A's estimate is shown to B, B's estimate is shown to C, ... E's estimate is shown to F. This was repeated across 15 rounds. A (head of chain) always experienced difficultu level 1, and all other participants in the chain experinced difficulty level 3.
```
Comment (sarikasu) [inline tasks/random_dot_motion.md:22]: Similar comment to the one above, should this be broken out into steps: 
1. Six people are given specific letters (A, B, C, D, E, F) in a chain 
2. Each person in the chain is paired with their predecessor for 15 consecutive rounds and the level of difficulty increases further down the chain (A-head of the chain always has difficulty level one and difficulty increases along the chain)
3. The individual at position k observed the final estimate of the individual at position k ‒ 1 before entering their final estimate

File: tasks/random_dot_motion.md [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+> Complete instructions shown to participants at each stage of the task.  
+> Write the complete instructions used from the start to the end of the experiment. This should include any instruction provided on each of the steps. If instructions have step interdependence, or are role specific, describe that in full detail.
+> 1. While the dots are still moving, participants indicated their estimate by using the computer mouse to place a black arrow in the circular area.
+> 2. After confirming the estimate (by a mouse click), the dots stopped moving and a blue arrow indicating the partner's guess was displaced. Participants were instructed to revise (or not revise) their estimate, and confirm their new estimate.
+> 3. The correct angle was displaced as a red arrow. 
+
+
+# Criteria
+## Performance calculation
+> A points system was used to reward more accurate guesses. The function $p=10^{-5e_f^2}$, where $e_f$ is the the error on the final estimate in radians, was rounded to the nearest integer.
```
Comment (sarikasu) [inline tasks/random_dot_motion.md:40]: Should we add in the range of points they could get? Participants could score between 0-10 points per round.

File: tasks/random_dot_motion.md [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+> 2. After confirming the estimate (by a mouse click), the dots stopped moving and a blue arrow indicating the partner's guess was displaced. Participants were instructed to revise (or not revise) their estimate, and confirm their new estimate.
+> 3. The correct angle was displaced as a red arrow. 
+
+
+# Criteria
+## Performance calculation
+> A points system was used to reward more accurate guesses. The function $p=10^{-5e_f^2}$, where $e_f$ is the the error on the final estimate in radians, was rounded to the nearest integer. 
+
+## Incentives
+> 0.005 Euros per point were awarded to participants.
```
Comment (sarikasu) [inline tasks/random_dot_motion.md:43]: They received a flat fee of 10.0 Euros for their participation, in addition to a monetary bonus depending on
the performance which you have included

File: tasks/random_dot_motion.md [hunk 4]
```diff
@@ -0,0 +1,43 @@
+# Summary
+> Participants are asked to answer which direction (0 to 360 degrees) a circular field of 50 dots are moving.
+
+# References
+> Dynamics of judgment propagation. Mehdi Moussaïd, Stefan M. Herzog, Juliane E. Kämmer, Ralph Hertwig. Proceedings of the National Academy of Sciences Apr 2017, 114 (16) 4117-4122; DOI: 10.1073/pnas.1611998114
+
```
Comment (sarikasu) [inline tasks/random_dot_motion.md:6]: Supporting Information: https://www.pnas.org/content/pnas/suppl/2017/03/29/1611998114.DCSupplemental/pnas.201611998SI.pdf?targetid=nameddest%3DSTXT

**Review by vivianxdx — APPROVED** [review]

**Review by vivianxdx — APPROVED** [review]

**Review by vivianxdx — APPROVED** [review]

**Review by vivianxdx — APPROVED** [review]

**Review by KaranSampath — APPROVED** [review]

**Pull request merged**